lib/Exporter/Writer/MimeTypedWriterInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Writer;
+
+interface MimeTypedWriterInterface extends WriterInterface
+{
+    /**
+     * There can be several mime types for a given format, this method should
+     * return the most appopriate / popular one.
+     *
+     * @return string the mime type of the output
+     */
+    public function getMimeType();
```

Comment:
> Default would mean you can set another, which might not be true… let's keep it simple ok? getMimeType really means "get the mime type we feel is the most appropriate", so I'll just add that as a description for the method.

I'm sorry but this should not be changed, you may set children methods final. :stuck_out_tongue: 

As I show to you, json for example can have multiple mime type, this is why I'm suggesting `Default`.